web/src/components/PoiDetails.tsx

```diff
@@ -174,29 +158,10 @@ const PoiDetails = ({ poi, distance, toolbar }: PoiDetailsProps): ReactElement =
           <StyledExternalLinkIcon src={ExternalLinkIcon} directionDependent />
         </StyledLink>
       </DetailSection>
-      {(!!website || !!phoneNumber || !!email) && (
-        <>
-          <Spacer $borderColor={theme.colors.borderColor} />
-          <Collapsible title={t('contactInformation')}>
-            <div>
-              {!!website && (
-                <ContactItem iconSrc={WebsiteIcon} iconAlt={t('website')} link={website} content={website} />
-              )}
-              {!!phoneNumber && (
-                <ContactItem
-                  iconSrc={PhoneIcon}
-                  iconAlt={t('phone')}
-                  link={`tel:${phoneNumber}`}
-                  content={phoneNumber}
-                />
-              )}
-              {!!email && (
-                <ContactItem iconSrc={MailIcon} iconAlt={t('eMail')} link={`mailto:${email}`} content={email} />
-              )}
-            </div>
-          </Collapsible>
-        </>
-      )}
+      {contacts.length > 0 &&
+        contacts.map(contact => (
+          <Contact key={contact.headline ?? contact.website ?? contact.name ?? contact.phoneNumber} contact={contact} />
+        ))}
```

Comment:
:wrench: Same as for native
```suggestion
      {contacts.map(contact => (
          <Contact key={contact.headline ?? contact.website ?? contact.name ?? contact.phoneNumber} contact={contact} />
        ))}
```